app/Http/Controllers/ChatStreamController.php

```
@@ -34,30 +104,131 @@ public function __invoke(ChatStreamRequest $request, Chat $chat): StreamedRespon
+
+                        // Stream Laravel model tool output directly as text
+                        if ($toolName === 'generate_laravel_model' && ! str_starts_with($result, 'Error:')) {
+                            yield json_encode([
+                                'type' => 'text',
+                                'content' => "\n\n".$result,
+                            ])."\n";
+                            $text .= "\n\n".$result;
+                        }
+
+                        // Stream knowledge search results
+                        if ($toolName === 'search_knowledge' && preg_match('/\[knowledge:(\d+) results\]/', $result, $kMatches)) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$kMatches'. (undefined)

(UnusedLocalVariable)